dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -353,29 +353,5 @@ public void setDbUpdateDate(Timestamp dbUpdateDate) {
 
     public enum ReferenceType { COMMIT, TAG, BRANCH, NOT_APPLICABLE, UNSET }
 
-    @Override
```

Comment:
You're calling super.hashCode() from Tag and WorkflowVersion; you need to keep these.